Doc/library/uuid.rst

```diff
@@ -330,8 +346,8 @@ The following options are accepted:
    Specify the function name to use to generate the uuid. By default :func:`uuid4`
    is used.
 
-   .. versionadded:: 3.14
-      Allow generating UUID version 8.
+   .. versionchanged:: next
+      Allow generating UUID versions 6 and 8.
```

Comment:
```suggestion
      Allow generating UUID versions 6-8.
```

Reply:
I find this less clear!

Reply:
Less clear than "Allow generating UUID versions 6, 7 and 8"?

I guess that one's okay like that, it's not many digits (unlike the others).

Reply:
Yes, less clear than writing the versions. In other languages (or contexts) we use prepositions to denote ranges, not hyphens.